util/chplenv/chpl_3p_qthreads_configs.py

```diff
@@ -11,9 +11,13 @@ def get_uniq_cfg_path():
 
 @memoize
 def get_link_args():
-    return third_party_utils.default_get_link_args('qthread',
-                                                   ucp=get_uniq_cfg_path(),
-                                                   libs=['libqthread_chpl.la',
-                                                         '-lchpl',
-                                                         'libqthread.la',
-                                                         '-lrt'])
+    link_args = \
+        third_party_utils.default_get_link_args('qthread',
+                                                ucp=get_uniq_cfg_path(),
+                                                libs=['libqthread_chpl.la',
+                                                      '-lchpl',
+                                                      'libqthread.la'])
+    if ( chpl_compiler.get('target').startswith('cray-prgenv') and
+         chpl_llvm.get() == 'llvm' ):
```

Comment:
Do you think `chpl_llvm.get() != 'none'` is more future proof, e.g. if we ever support CHPL_LLVM=system?. No need to change it now, I am just curious.